getting-started-guides/csp/gcp/spark-gpu/README.md

```diff
@@ -11,56 +11,99 @@ Prerequisites
+
+Execute the commands below while in the spark-examples folder.  These commands will copy the following files into your GCP bucket: 
+
+1. Initialization scripts for GPU and RAPIDS Spark, 
+2. PySpark app files  
+3. A sample dataset for a XGBoost PySpark app 
+4. The latest Spark RAPIDS XGBoost jar files from the public maven repository
+
+```bash
+export GCS_BUCKET=my-bucket
+export RAPIDS_SPARK_VERSION='2.x-1.0.0-Beta3'
+export RAPIDS_CUDF_VERSION='0.9.2-cuda10'
+pushd getting-started-guides/csp/gcp/spark-gpu
+gsutil cp -r internal gs://$GCS_BUCKET/spark-gpu/
+gsutil cp rapids.sh gs://$GCS_BUCKET/spark-gpu/rapids.sh
+popd
+pushd datasets/
+tar -xvf mortgage-small.tar.gz
+gsutil cp -r mortgage-small/ gs://$GCS_BUCKET/
+popd
+wget -O cudf-${RAPIDS_CUDF_VERSION}.jar https://repo1.maven.org/maven2/ai/rapids/cudf/${RAPIDS_CUDF_VERSION%-*}/cudf-${RAPIDS_CUDF_VERSION}.jar
+wget -O xgboost4j_${RAPIDS_SPARK_VERSION}.jar https://repo1.maven.org/maven2/ai/rapids/xgboost4j_${RAPIDS_SPARK_VERSION/-/\/}/xgboost4j_${RAPIDS_SPARK_VERSION}.jar
+wget -O xgboost4j-spark_${RAPIDS_SPARK_VERSION}.jar https://repo1.maven.org/maven2/ai/rapids/xgboost4j-spark_${RAPIDS_SPARK_VERSION/-/\/}/xgboost4j-spark_${RAPIDS_SPARK_VERSION}.jar
+gsutil cp cudf-${RAPIDS_CUDF_VERSION}.jar xgboost4j-spark_${RAPIDS_SPARK_VERSION}.jar xgboost4j_${RAPIDS_SPARK_VERSION}.jar gs://$GCS_BUCKET/
+````
+
+After executing these commands, use your web browser to navigate to Google Cloud Platform console and make sure your Google storage bucket “my-bucket” directory structure has the following files:
+* gs://my-bucket/spark-gpu/rapids.sh
+* gs://my-bucket/spark-gpu/internal/install-gpu-driver-debian.sh
+* gs://my-bucket/spark-gpu/internal/install-gpu-driver-ubuntu.sh
+* gs://my-bucket/cudf-${RAPIDS_CUDF_VERSION}.jar
+* gs://my-bucket/xgboost4j-spark_${RAPIDS_SPARK_VERSION}.jar
+* gs://my-bucket/xgboost4j_${RAPIDS_SPARK_VERSION}.jar
+* gs://my-bucket/mortgage-small/eval/mortgage-small.csv
+* gs://my-bucket/mortgage-small/eval/mortgage-small.csv
+* gs://my-bucket/mortgage-small/trainWithEval/test.csv
+
+
+### Step 2 Using the `gcloud` command to create a new cluster with this initialization action. 
+
+The following command will create a new cluster named `<CLUSTER_NAME>`. Before the init script fully merged into 
+`<dataproc-initialization-actions>` bucket, user need to copy the sparkgpu initialization script into a accessible GCS and into following structure. Ubuntu is recommended as CUDA support ubuntu, debian could be used by modifying `image-version` and `linux-dist` accordingly.
 
+    ```bash
+    /$GCS_BUCKET/sparkgpu/rapids.sh
```

Comment:
Is "sparkgpu" correct?
Some places above use "spark-gpu".

Reply:
Good catch, I have fixed it.